### PR TITLE
Add more company logos

### PR DESCRIPTION
**Pull request by discordianfish**
- DigitalOcean
- Docker
- GiantSwarm
- Jodel
- Quobyte
- Weave

### CODE REVIEW

**Comment by juliusv** [comment]
Other potential ones:

- Walmart
- Monsanto
- Matrix.org

There's many others of course...

**Comment by fabxc** [comment]
https://twitter.com/SeastarIO/status/692057490268123137

**Comment by brian-brazil** [comment]
Please keep the logos sorted.

SeaStar have confirmed they'd like to be listed. Talking to Walmart previously, they asked me not to list them.

**Comment by grobie** [comment]
Please let's not list Monsanto.

**Comment by discordianfish** [comment]
@grobie: I'm not sure.. where would you draw the line? Would we accept contributions from them? Do we accept them using Prometheus as name? (I gues we can't really do anything about it but still..).
And about other questionable institutions?

I'm not saying we should put them up there but we should probably figure out where to draw the line. I was always thinking it would be great to have some kind of manifesto for engineers or projects to follow in such cases..

**Comment by juliusv** [comment]
We could have this discussion about other companies as well ;) Anyways, not urgent.

Btw., we just got permission to use the Weave (http://www.weave.works/) logo!

**Comment by discordianfish** [comment]
I've added weave although maybe we can find a nicer logo.

File: content/index.html [hunk 1]
```diff
@@ -108,6 +114,8 @@
       <div class="col-md-2 col-sm-4 col-xs-6 logos">
         <a href="https://www.variomedia.de/"><img src="assets/company-logos/variomedia.png"/></a>
       </div>
-    </div>
+      <div class="col-md-2 col-sm-4 col-xs-6 logos">
+        <a href="https://www.weave.works/"><img src="assets/company-logos/weave.png"/></a>
```
Comment (juliusv) [inline content/index.html]: Did you check this in?

Comment (discordianfish) [inline content/index.html]: Missed it, it's in now.

Comment (fabxc) [inline content/index.html]: Why the one with the black background?

Comment (discordianfish) [inline content/index.html]: @fabxc As I said, it's the only one I could find.

Comment (fabxc) [inline content/index.html]: I found this one: http://www.ricston.com/wp-content/uploads/2015/07/P-weaveWorks_colour_logo_POS_RGB_320x132.png

We could certainly asked them for one, but the size seems sufficient.

**Comment by discordianfish** [comment]
Okay everyone, this is ready to get merged.

**Comment by grobie** [comment]
:+1: If we have all the permissions. Thanks for the work!

**Comment by juliusv** [comment]
:+1: awesome

**Pull request merged**